code/drasil-example/template/lib/Drasil/Template/Body.hs

```diff
@@ -141,3 +181,191 @@ progName = commonIdeaWithDict "progName" (pn "ProgName") "ProgName" []
 -- MOVE TO DATA.PEOPLE
 authorName :: Person
 authorName = person "Author" "Name"
+
+{-------------------------------------------------------------------------------
+- Equilibrium
+-------------------------------------------------------------------------------}
+
+equilibrium :: IdeaDict
+equilibrium = nc "equilibrium" $ nounPhrase "equilibrium" "equilibria"
+
+-- There is a generic copy between economics and physics, but we won't bother
+-- with that.
+
+{-------------------------------------------------------------------------------
+- Equilibrium (Economics)
+-------------------------------------------------------------------------------}
+
+equilibriumQ :: QuantityDict
+equilibriumQ = mkQuant' -- A somewhat odd variant of `mkQuant` that re-orders argument and adds staging to the `Symbol`
+  "inEquilibrium"
+  (nounPhrase "in equilibrium" "all equilibria") -- duplicates the IdeaDict
+  Nothing -- UnitDefn
+  Boolean
+  (autoStage $ label "inEquilibrium")
+  Nothing -- "abbreviation String"
+
+price :: IdeaDict -- I guess making this is completely unnecessary because the QuantityDict will make its own.
+price = nc "price" $ cn "price"
+
+priceQ :: QuantityDict
+priceQ = mkQuant'
+  "priceQuant" -- duplicates the IdeaDict
```

Comment:
It is technically a duplication -- but it shouldn't be, because the 'concept' of a price, and a variable that holds a 'price' value are completely different. And the bug is that you shouldn't do `(cn "price")` below but refer to `price ^. term` instead.